.circleci/config.yml

```diff
@@ -683,6 +684,24 @@ jobs:
                 --target-slug mapbox/mapbox-maps-android-private
             fi
 
+  check-gradle-denendencies:
+    executor:
+      name: ubuntu
+      machine: small
+    steps:
+      - checkout
+      - restore-gradle-cache
+      - run:
+          name: Run dependency check
+          no_output_timeout: 15m
+          command: ./gradlew dependencyUpdates --stacktrace
```

Comment:
Do we want to keep `--stacktrace` or is it just for testing?

Reply:
remove it